src/payment/spontaneous.rs

```diff
@@ -58,6 +58,27 @@ impl SpontaneousPayment {
 	/// node-wide parameters configured via [`Config::sending_parameters`] on a per-field basis.
 	pub fn send(
 		&self, amount_msat: u64, node_id: PublicKey, sending_parameters: Option<SendingParameters>,
+	) -> Result<PaymentId, Error> {
+		self.send_inner(amount_msat, node_id, sending_parameters, None)
+	}
+
+	/// Send a spontaneous payment including a custom TLV key and value.
+	pub fn send_with_custom_tlv(
+		&self, amount_msat: u64, node_id: PublicKey, sending_parameters: Option<SendingParameters>,
+		custom_tlv_type: u64, custom_tlv_value: Vec<u8>,
```

Comment:
Our implementation uses a Vec of TlvEntry, it would be great if it worked similar (a separate method is also fine if that is preferred):

https://github.com/getAlby/ldk-node/blob/main/src/payment/spontaneous.rs#L59

https://github.com/getAlby/ldk-node/blob/main/bindings/ldk_node.udl#L159

Reply:
nice, I took a similar approach